languages/fsharp/exercises/concept/arrays/.docs/introduction.md

```diff
@@ -0,0 +1,38 @@
+An `array` in F# is a mutable collection of zero or more values with a fixed length. This means that once an array has been created, its size cannot change, but its values can. The values in an array must all have the same type. Arrays can be defined as follows:
+
+```fsharp
+let empty = [| |]
+let emptyAlternative = Array.empty
+
+let singleValue = [| 5 |]
+let singleValueAlternative = Array.singleton 5
+
+let threeValues = [| "a"; "b"; "c" |]
+```
+
+Elements can be assigned to an array or retrieved from it using an index. F# arrays are zero-based, meaning that the first element's index is always zero:
+
+```fsharp
+let numbers = [2; 3; 5]
```

Comment:
Just looking through this PR for inspiration, but noticed this, should it be:

```suggestion
let numbers = [| 2; 3; 5 |]
```

Reply:
Absolutely! Thanks